src/components/slack/attachment/attachment.component.ts

```diff
@@ -11,6 +11,7 @@ import { SettingService } from '../../../services/setting.service';
 export class SlackAttachmentComponent {
     @Input() attachment: Attachment;
     @Input() parser: SlackParser;
```

Comment:
Attachment field value and footer text are using `parser` method yet. I don't know we can replace these parser.

https://github.com/KongariBug/ASlack-Stream/blob/f7e071f1c58212f77d492c2dfee3b689c6881683/src/components/slack/attachment/attachment.component.html#L38
https://github.com/KongariBug/ASlack-Stream/blob/f7e071f1c58212f77d492c2dfee3b689c6881683/src/components/slack/attachment/attachment.component.html#L48